torch/quantization/fx/quantize.py

```diff
@@ -955,6 +967,14 @@ def record_match(pattern, node, matched):
         return match_map
 
     def _find_quants(self, graph, matches):
+        """
+        Inputs:
+          - graph: an fx.Graph object
+          - matches: output of self._find_matches function
+
+        Outputs a map of
+          node_name -> (DefaultQuant instance, qconfig)
```

Comment:
QuantizeHandler instance would be better I think

Reply:
what would be the context for that?  Seems like only `DefaultQuant` instances are added in this function

Reply:
DefaultQuant is inheriting from QuantizeHandler, QuantizeHandler occurred in other places as well so it would be clearer to use it here I think. we can roughly think about QuantizeHandler as the type and DefaultQuant as specific instance

Reply:
basically find_matches find QuantizeHandler for quantizable ops and find_quants find QuantizeHandler for inputs/outputs of these ops